### PR TITLE
Update GitHub OpenAPI with idiomatic namings

### PR DESCRIPTION
**Pull request by MahdiBM**

### CODE REVIEW

File: Lambdas/GitHubAPI/openapi-generator-config.yml [hunk 1]
```diff
@@ -2,3 +2,4 @@ generate:
   - types
   - client
 accessModifier: package
+namingStrategy: idiomatic
```
Comment (MahdiBM) [inline Lambdas/GitHubAPI/openapi-generator-config.yml:5]: The new setting

**Review by gwynne — APPROVED** [review]
Not sure what problem this actually solves, but sure, go ahead 😆

File: Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift [hunk 2]
```diff
@@ -65,28 +65,28 @@ struct DocsIssuer {
     }
 
     func getPRsRelatedToCommit() async throws -> [SimplePullRequest] {
-        try await context.githubClient.repos_list_pull_requests_associated_with_commit(
+        try await context.githubClient.reposListPullRequestsAssociatedWithCommit(
             path: .init(
                 owner: self.repo.owner.login,
                 repo: self.repo.name,
-                commit_sha: self.commitSHA
+                commitSha: self.commitSHA
```
Comment (0xTim) [inline Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift:72]: Now we just need to capitalise this properly 😛

**Review by 0xTim — APPROVED** [review]

File: Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift (hunk 2 above)
Comment (MahdiBM) [reply Lambdas/GHHooks/EventHandler/Handlers/DocsIssuer.swift:72]: I don't think it's possible, automatically, but the generator does take custom key value name mapping pairs and we could use that although I don't care enough for that.

**Pull request merged**